client/src/components/pages/HomeWork/data.js

```diff
@@ -0,0 +1,15 @@
+const data = [
+  {
+    id: 1,
+    Homework_description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took',
+    HmURLS: [
```

Comment:
elements Array needs to unique id , because when you render it, you will needed.
Like: array of object
HumURLS:[
{id: 1, link: 'https://www.npmjs.com/package/prop-types'},
...
]

Reply:
Done